deployment/docker/REQUIREMENTS-dev.txt

```diff
@@ -0,0 +1,13 @@
+django-nose
+coverage
+pep8>=1.5.7,<1.6
+pylint
+flake8==3.3.0
+factory_boy
```

Comment:
I'm wondering why we would include some of the requirements here (like `factory_boy`) and not in production mode?

Reply:
This is for testing purpose, that's why it's on the dev.

Reply:
Wouldn't we ever run tests in production mode on something like travis?

Reply:
In travis we are using dev mode.

Reply:
Yeah tests only run on the dev mode, and travis gets its own settings profile for testing.